rumqttc/README.md

```diff
@@ -77,17 +77,25 @@ Quick overview of features
 
 In short, everything necessary to maintain a robust connection
 
-**NOTE**: Looping on `connection.iter()/eventloop.poll()` is necessary to
-run the eventloop and make progress. It yields incoming and outgoing activity
-notifications which allows customization as user sees fit.
-
-**IMPORTANT** Blocking inside `eventloop.poll()/eonnection.iter()` loop
-will block connection progress
-
 Since the eventloop is externally polled (with `iter()/poll()` in a loop)
 out side the library and `Eventloop` is accessible, users can
 - Distribute incoming messages based on topics
 - Stop it when required
 - Access internal state for use cases like graceful shutdown or to modify options before reconnection
 
+## Important notes
+
+- Looping on `connection.iter()/eventloop.poll()` is necessary to run the
+  event loop and make progress. It yields incoming and outgoing activity
+  notifications which allows customization as you see fit.
+
+- Blocking inside the `eventloop.poll()/eonnection.iter()` loop will block
```

Comment:
There is a typo in "connection" here, but I suggest you switch the two just to be consistent with the note above.

```suggestion
- Blocking inside the `connection.iter()`/`eventloop.poll()` loop will block
```